-------------------------------------------------------------------------------
Test set: altunity.alttrashcatj.MainMenuTests
-------------------------------------------------------------------------------
Tests run: 2, Failures: 0, Errors: 2, Skipped: 0, Time elapsed: 2.424 s <<< FAILURE! - in altunity.alttrashcatj.MainMenuTests
altunity.alttrashcatj.MainMenuTests  Time elapsed: 2.424 s  <<< ERROR!
ro.altom.altunitytester.altUnityTesterExceptions.ConnectionException: Could not create connection to 192.168.11.209:13000
	at altunity.alttrashcatj.MainMenuTests.setUp(MainMenuTests.java:22)
Caused by: java.net.ConnectException: Connection refused: connect
	at altunity.alttrashcatj.MainMenuTests.setUp(MainMenuTests.java:22)

altunity.alttrashcatj.MainMenuTests  Time elapsed: 2.424 s  <<< ERROR!
java.lang.NullPointerException
	at altunity.alttrashcatj.MainMenuTests.tearDown(MainMenuTests.java:33)

